Source/Scene/Model.js

```diff
@@ -686,6 +686,13 @@ define([
         this._useDefaultSpecularMaps = false;
 
         this._shouldRegenerateShaders = false;
+        this._supportsWebp = undefined;
+
+        var that = this;
+        FeatureDetection.supportsWebp()
```

Comment:
I think I mentioned this before, but just in case, this can't be done here; it's a race condition.  We need to actually perform this check at load time so that it's part of the async chain.

Reply:
Are you sure this is still a race condition? I've moved this from happening on module load to in the constructor. Model.prototype.update will wait until `_supportsWebp` is defined. https://github.com/AnalyticalGraphicsInc/cesium/pull/7486/files#diff-bcedb35459e5bed46206bad4990bc7f1R4311

I think if I add an `.otherwise(function(){ that._supportsWebp = false })` it'll be fool-proof then.

Reply:
See https://github.com/AnalyticalGraphicsInc/cesium/pull/7486#discussion_r252742070